responsive-article/gulcin/index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Medium Article</title>
+  <link href="styles.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/8a55a6124d.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Pathway+Extreme:wght@400;600&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="left-nav">
+        <div class="medium-icon">
+          <a href="#"><img src="images/iconn.jpg" alt="Medium Icon"></a>
+        </div>
+        <div class="search">
+          <i class="fa-solid fa-magnifying-glass"></i>
+          <label for="nav-search"></label>
```

Comment:
If you don't have any label text for the input, then you don't really need to add label element.
For accessibility, it's still good to have a label or a replacement though.
There are a few ways to do it:
- Add aria-label attribute directly to input `<input type="search" id="nav-search" name="nav-search" placeholder="Search Medium" aria-label="Search Medium">`
- Add the text you want to the label element but then move the label away from the screen  ```.off-screen { position: absolute;
    left: -9999px;
}```. 

My personal opinion is that the first approach is nicer.